dependencies.xml

```diff
@@ -41,7 +41,7 @@ Note all install methods after "main" take
     <statsmodels/>
     <cloudpickle>1.6</cloudpickle>
     <tensorflow>1.15</tensorflow>
-    <python skip_check='True'>3</python>
+    <python skip_check='True'>3.7</python>
```

Comment:
Do we all agree with this change?

Reply:
I think we should figure out the issue. It seems it just caused by this PR.

Reply:
I reported this already, but it seems that with Python 3.8 and 3.9 the library versions are not compatible

Reply:
@aalfonsi FYI, from Josh's diagnosis:
```
failing:
numpy-1.18.5
xarray-0.16.2
Python: 3.6.9
Passing:
numpy-1.18.5
xarray-0.16.2
Python: 3.7.10

Okay, so we know that it is something that changes between 3.6 and 3.7, and that 3.6 is reasonably consistent.  
Here is the list of changes: https://docs.python.org/3/whatsnew/3.7.html
If I had to guess, the change that is causing the problem is:
the insertion-order preservation nature of dict objects has been declared to be an official part of the Python language spec.

Possibly you need to change a dict to and ordered dictionary (Before 3.7, dict and collections.OrderedDict are different)
```

Reply:
3.7 was an attempt to fix a test, the test was removed so I reverted back to 3